source/api_documentation/network_integration/customer_phone_numbers/index.rst

```diff
@@ -0,0 +1,110 @@
+Destinations
+============
+
+Manage destinations (aka Customer Phone Numbers)
+""""""""""""""""""""""""""""""""""""""""""""""""
+
+Destinations are alternate way of managing :doc:`customer_phone_numbers <../ringpools/index>`.
+
+`Learn more about RingPools here <https://community.invoca.com/t5/call-attribution/basic-knowledge-ringpools/ta-p/530>`_.
+
+.. list-table::
+  :widths: 11 34 40
+  :header-rows: 1
+  :class: parameters
+
+  * - Property
+    - Type
+    - Value
+
+  * - id
+    - integer (read-only)
+    - The internal Invoca id of the Destination.
+
+  * - phone_number
+    - string
+    - Phone number of the Destination.
+
+  * - primary_url
+    - string
+    - ToDo
+
+  * - requested_destination_name
+    - integer
+    - ToDo
```

Comment:
https://github.com/Invoca/developer-docs/pull/340#pullrequestreview-1688509027 Cont.

Not sure what `primary_url` or `requested_destination_name` mean. I couldn't find a UI input related to it either. So we can either remove these from here and API as well or I can put any description that you have in mind.

Reply:
Yeah, there is some awkwardness in trying to find how these values map to the frontend.  

`primary_url` is the also referred to as "URL Found On" or something like that.  It marks the exact URL that the number can be found on the customer's website.  We don't really process it for anything as far as I'm aware, but the Destination Settings can be configured to allow our tag to capture CustomerPhoneNumbers automatically, in which case, we capture the `primary_url` for them, so they know where to find it if they're interested in knowing how it was generated.  

`request_destination_name` is just the "Name" or "Friendly Name" of a CustomerPhoneNumber, just an alias for identifying them beyond the phone number itself.

![Screenshot 2023-10-23 at 11 38 14 AM](https://github.com/Invoca/developer-docs/assets/17100108/f6107ead-57ce-4812-8621-6f76cce68a98)

In my above screenshot, the URL column is empty since I manually entered them through the UI but that _would_ show the URL that the number was one if their tag was setup to detect & create destinations automatically.  In this context, I think it correct to return both values, even if they might not be that useful for FindLaw, just because it's appropriate for broad usage.